Jenkinsfile

```diff
@@ -374,20 +360,13 @@ pipeline {
           golang(){
             dir("${BASE_DIR}"){
               sh(label: 'Build packages', script: './script/jenkins/package.sh')
-              sh(label: 'Test packages install', script: './script/jenkins/test-install-packages.sh')
```

Comment:
Why do we remove the test?

Reply:
forget, I've seen it is only for testing

Reply:
For the record, I just wanna be sure by reusing the same agent between stages, even when running some docker magic steps the permissions are intact, therefore no surprises for those stages which are only executed in the master branch.